app/elements/calendar-app.html

```diff
@@ -109,7 +113,7 @@
               height: 'auto',
               firstDay: 1,
               columnFormat: 'ddd D/M',
-              titleFormat: "([week] W), MMM D YYYY",
+              titleFormat: "([week] W)",
```

Comment:
Fullcalendar doesn't seem to allow for it, it's defaulting to `{{titleFormat}} - {{titleFormat}}`